<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTCPClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main_page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTabWidget" name="mainTabWidget">
          <property name="styleSheet">
           <string notr="true">#mainTabWidget QTabWidget::pane {
    background-color: white;
    border: 2px solid black;
	border-radius:8px
}

#mainTabWidget QTabBar::tab {
    background: white;
    border: 3px solid black;
    border-radius: 8px; 
    padding: 10px;
	margin-right: 10px;
	margin-bottom: 5px;
	min-width: 100px;  
    min-height: 30px;  
	color: black;
	font-size: 16px;
}

#mainTabWidget QTabBar::tab:selected {
    background: gray;
	color: white;
	font-weight: bold;
    border-radius: 5px;
}

QTabWidget::tab-bar {  /* 탭 버튼을 감싸는 바 */
    left: 20px;  /* 탭 목록이 왼쪽으로 조금 이동 */
}</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="entireListTab">
           <attribute name="title">
            <string>웹툰 전체목록</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableView" name="e_tableView">
              <property name="font">
               <font>
                <family>Ubuntu</family>
                <pointsize>13</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <property name="textElideMode">
               <enum>Qt::TextElideMode::ElideMiddle</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::PenStyle::SolidLine</enum>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="dayTab">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>요일별 웹툰</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTabWidget" name="dayTabWidget">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">#dayTabWidget QTabWidget::pane {
    background-color: white;
    border: 2px solid black;
	border-radius:8px
}

#dayTabWidget QTabBar::tab {
    background: lightblue;
    border: 3px solid black;
    border-radius: 8px; 
    padding: 10px;
	margin-right: 10px;
	margin-bottom: 5px;
	min-width: 65px;  
    min-height: 20px;  
	color: black;
	font-size: 16px;
}

#dayTabWidget QTabBar::tab:selected {
    background: darkblue;
	color: white;
	font-weight: bold;
    border-radius: 5px;
}

QTabWidget::tab-bar {  /* 탭 버튼을 감싸는 바 */
    left: 20px;  /* 탭 목록이 왼쪽으로 조금 이동 */
}</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_5">
               <attribute name="title">
                <string>월요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_mon">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_6">
               <attribute name="title">
                <string>화요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_tue">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_7">
               <attribute name="title">
                <string>수요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_wed">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_8">
               <attribute name="title">
                <string>목요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_thu">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_9">
               <attribute name="title">
                <string>금요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_fri">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_10">
               <attribute name="title">
                <string>토요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_sat">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_11">
               <attribute name="title">
                <string>일요일</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableView_sun">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="searchTab">
           <attribute name="title">
            <string>웹툰 검색</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="searchComboBox">
                <item>
                 <property name="text">
                  <string>제목</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>작가</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="toonSearchText"/>
              </item>
              <item>
               <widget class="QPushButton" name="searchButton">
                <property name="text">
                 <string>검색</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="searchTableView">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="bookMarkTab">
           <attribute name="title">
            <string>웹툰 즐겨찾기</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTableView" name="b_tableView">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="toonList_page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QTableView" name="epiList_tableView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="1">
            <widget class="QPushButton" name="toList_backBtn">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>뒤로 가기</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="webtoon_page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_16">
          <item row="0" column="0">
           <widget class="QScrollArea" name="toon_scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="toon_scrollArea_contents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>762</width>
               <height>488</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4"/>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="1" column="1">
           <widget class="QPushButton" name="like_btn">
            <property name="text">
             <string>좋아요</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="beforeEpi_btn">
            <property name="text">
             <string>&lt;&lt;이전화</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="backList_btn">
            <property name="text">
             <string>목록으로 돌아가기</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="afterEpi_btn">
            <property name="text">
             <string>다음화&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="bookmark_add_btn">
            <property name="text">
             <string>즐겨찾기 추가</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>735</width>
          <height>58</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_sendMessage">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Send Message</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_sendAttachment">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Send Attachment</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit_message">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_receivedMessages">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>16</y>
          <width>735</width>
          <height>358</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>toonSearchText</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
